packages/origin-ui-core/src/components/Modal/AskDetailslModal.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Dialog } from '@material-ui/core';
+import { Order } from '../../utils/exchange';
+
+interface IOwnProps {
+    ask: Order;
+    close: () => void;
+}
+
+export const AskDetailsModal = (props: IOwnProps) => {
+    const { ask, close } = props;
+    return (
+        <Dialog open={ask !== null} onClose={close}>
+            View ask details modal
```

Comment:
Needs to be localized.

Reply:
These modals (Ask and Bids details and remove) are templates and developed in other PR